packages/calypso-e2e/src/lib/flows/change-ui-language-flow.ts

```diff
@@ -0,0 +1,43 @@
+import { Page } from 'playwright';
+import { NavbarComponent, SidebarComponent } from '../components';
+import { AccountSettingsPage } from '../pages';
+
+/**
+ * Change the UI language.
+ */
+export class ChangeUILanguageFlow {
+	private page: Page;
+
+	/**
+	 * Constructs an instance of the flow.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 * Changes the UI langauge.
+	 *
+	 * This method will navigate from the navbar to the /me page,
+	 * then onto Account Settings.
+	 */
+	async changeUILanguage( localeSlug: string ): Promise< void > {
+		await new SidebarComponent( this.page ).waitForSidebarInitialization();
```

Comment:
You are right, it turned out not to be needed. I've added it initially as I was working fixing a flaky step and it seemed like it fixed it, but it might've been just a fluke.